examples/create_reference.py

```diff
@@ -170,7 +170,7 @@ def main():
 
   client.close()
   if reference_obj:
-    print(json.dumps(reference_obj.to_dict(), indent=2))
+    print(pprint(reference_obj.to_dict()))
```

Comment:
No need to call print here
```suggestion
    pprint(reference_obj.to_dict())
```